### ISSUE TITLE
'ngcc is already running' when running 'yarn dev'

### ISSUE THREAD

**Issue by ranfdev** [Chocobozzz/PeerTube#2508]
When trying to run this in gitpod i get the following error:

```
ERROR in ngcc is already running at process with id 70.
If you are running multiple builds in parallel then you should pre-process your node_modules via the command line ngcc tool before starting the builds;
See https://v9.angular.io/guide/ivy#speeding-up-ngcc-compilation.
```
This is also happens on my machine when i try to run the docker image built from `./support/docker/dev/Dockerfile` (i edited just the last line of the file, else the container won't start). The error is from the command ```yarn dev```. From what i've read this can be solved by running the command `ngcc` after the installation of the npm packages.

If it works, i will send a pull request

**Comment by ranfdev** [comment]
So... i found the real issue:
When you run `npm run dev`, you are actually running the `scripts/dev/client.sh` and `scripts/dev/server.sh` scripts. When the `client.sh` script starts, it spawns an `ng serve` process, so there's a `ngcc` instance running and compiling the angular files. The first time `server.sh` is run, it will also run 
```bash
echo "client/dist/en-US/index.html does not exist, compile client files..."
npm run build:client -- --light
```
(line 6 of scripts/dev/server.sh).
the command `npm run build:client` actually spawns another `ngcc`. So there are 2 ngcc running. 
In angular 9 you can't run two ngcc at the same time. There's a lock file that prevents this. So everything crashes

TL;DR: the first time `npm run dev` is run, it will build the angular files two times, simultaneously
You can solve this issue by removing the lines from 5 to 9 in `scripts/dev/server.sh

Obviously removing lines at random isn't a great solution, but i don't know enough of the project to provide anything better.

**Comment by Chocobozzz** [comment]
Thanks for the issue. Should be fixed in https://github.com/Chocobozzz/PeerTube/commit/99ae9d3a9eeaf3db5c66bde006aded133757cb17
